app/views/notes/_comment.html.erb

```diff
@@ -73,15 +73,33 @@
   </div>
 
   <script>
-    // deletion button code:
-    $("#c<%= comment.cid %>delete-btn").bind('ajax:success', function(e,response){
-      $("#c<%= comment.cid %>").remove()
-      $('#comment-count')[0].innerHTML = parseInt($('#comment-count')[0].innerHTML)-1
-    })
-    $("#c<%= comment.cid %>delete-btn-2").bind('ajax:success', function(e,response){
```

Comment:
@jywarren I think this was redundant. What do you think?

Reply:
@ViditChitkara As you are working a lot on commenting i think you may have idea about this...is this redundant here?